src/utils/coingeckoApi.ts

```diff
@@ -27,3 +29,28 @@ export const fetchHistoricalTokenMarketData = (
     })
     .catch((error) => console.log(error))
 }
+
+export const fetchCurrentPrice = async (
+  address: string,
+  baseCurrency = 'usd'
+): Promise<number> => {
+  console.log(address)
+  if (address === 'ETH') {
+    console.log('tets')
+    const getPriceUrl =
+      baseURL + `/simple/price/?ids=ethereum&vs_currencies=${baseCurrency}`
+    const resp = await fetch(getPriceUrl)
+    const data = await resp.json()
+    if (!data['ethereum']) return 0
+    return data['ethereum'][baseCurrency]
+  }
+
+  const getPriceUrl =
+    baseURL +
+    `/simple/token_price/ethereum/?contract_addresses=${address}&vs_currencies=${baseCurrency}`
+
+  const resp = await fetch(getPriceUrl)
+  const data = await resp.json()
+  if (!data[address.toLowerCase()]) return 0
+  return data[address.toLowerCase()][baseCurrency]
+}
```

Comment:
remove console.logs. Also do we need try/catch blocks in here? if the coingecko request fails how does that affect the rest of the flow?